aii-ks/src/main/perl/ks.pm

```diff
@@ -692,13 +692,12 @@ sub log_action {
     }
 
     if ($consolelogging) {
-        push(@logactions, '# Make sure messages show up on the serial console',
-                          "tail -f $logfile > /dev/console &");
+        push(@logactions, "tee $logfile", "tee /dev/pts/0");
     }
 
-    push(@logactions,"drainsleep=$drainsleep"); # add trailing newline
-    push(@logactions,''); # add trailing newline
-    return join("\n", @logactions)
+#    push(@logactions,"drainsleep=$drainsleep"); # add trailing newline
+#    push(@logactions,''); # add trailing newline
+    return join(" | ", @logactions)
```

Comment:
the remote logging feature generates after exec
```
(tail -f /tmp/pre-log.log | awk '{print "<190>AII: "$0; fflush(); system("usleep 1000 >& /dev/null");}' | nc -u gastly.gastly.os 515) &
```
i'm not sure how to replace this more pipes